frontend/app/controllers/spree/orders_controller.rb

```diff
@@ -1,7 +1,6 @@
 module Spree
   class OrdersController < Spree::StoreController
     before_action :check_authorization
-    rescue_from ActiveRecord::RecordNotFound, with: :render_404
```

Comment:
I think it is very common in stores to overwrite `render_404` and do stuff instead of rendering the 404 page. Building a dynamic 404 page for instance. Is there a way to still provide this functionality for stores? Could you tell the dummy app where to find the views like you did with the assets paths for instance?

Reply:
I think this should be removed anyways because it's a bad way to override the 404 behaviour.

Stores could regain this functionality by either:
* Using `config.exceptions_app`, the proper way to have a dynamic 404 page
* Setting up the `rescue_from` from `ApplicationController`
* Class-evaling (or `send`ing) the rescue_from (if overrides are being done anyways)

But I can submit this as a separate PR if it will get this merged

Reply:
Right. I can't remove this because the render_404 behaviour is very unusual.

``` ruby
render status: :not_found, file: "#{::Rails.root}/public/404", formats: [:html], layout: nil
```

This doesn't use views, it renders the `public/404.html` file relative to the Rails.root directory. There's really no way to get around this, it's bad code.

Reply:
TIL `config.exceptions_app` 🎉 

You are right, the code is bad and should be removed. Although it's blocking your PR I think we need a separate PR due to visibility reasons. I think we can forego with deprecating this, though.